### PR TITLE
Fixing broken link

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: public/gcintranet/samples/custom-menu-en.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
 					"id": "link2",
 					"text": "Link 2",
 					"subLinks": [{
-						"subhref" : "https://domain.com",
+						"subhref" : "https://www.google.ca/",
```
Comment (nschonni) [inline public/gcintranet/samples/custom-menu-en.html:70]:
```suggestion
						"subhref" : "https://example.com/",
```

File: public/gcintranet/samples/custom-menu-en.html [hunk 2]
```diff
@@ -132,7 +132,7 @@ <h2>Implementation</h2>
         "id": "link2",
         "text": "Link 2",
         "subLinks": [{
-            "subhref" : "https://domain.com",
+            "subhref" : "https://www.google.ca/",
```
Comment (nschonni) [inline public/gcintranet/samples/custom-menu-en.html:135]:
```suggestion
            "subhref" : "https://example.com/",
```

**Review by nschonni — COMMENTED** [review]
https://en.wikipedia.org/wiki/Example.com is specifically setup for placeholder links

File: public/gcintranet/samples/custom-menu-en.html (hunk 1 above)
Comment (ahmad-shahid) [reply public/gcintranet/samples/custom-menu-en.html:70]: Yes that can be used but I prefer google for security and durability reasons.

**Review by plup2 — APPROVED** [review]

**Pull request merged**